spec/controllers/posts_controller_spec.rb

```diff
@@ -1998,16 +1998,65 @@
         expect(post.authors).to match_array([user, coauthor, joined_author])
       end
 
-      it "works as a coauthor" do
+      it "does not allow coauthors to edit post text" do
+        skip "Is not currently implemented on saving data"
         user = create(:user)
-        login_as(user)
+        coauthor = create(:user)
+        login_as(coauthor)
+        post = create(:post, user: user, authors: [user, coauthor], authors_locked: true)
+        put :update, params: {
+          id: post.id,
+          post: {
+            content: "newtext"
+          }
+        }
+        expect(response).to redirect_to(post_url(post))
+        expect(flash[:error]).to eq("You do not have permission to modify this post.")
```

Comment:
Needs a line to check content hasn't changed, but it's skipped anyway so this isn't major.